app/workers/ckan/v26/ckan_org_sync_worker.rb

```diff
@@ -2,6 +2,7 @@ module CKAN
   module V26
     class CKANOrgSyncWorker
       include Sidekiq::Worker
+      sidekiq_options queue: :sync
```

Comment:
This worker is scheduled daily, so does it also need a retry limit perhaps?

The table here https://github.com/mperham/sidekiq/wiki/Error-Handling suggests that 13 or 14 might be a good limit.

Reply:
This one syncs organisation metadata across from CKAN.  I wonder whether it should actually be run more often than daily?  Would a new publisher have to wait up to a day for their datasets to show on Find?

Reply:
14 would be 10 hours, right?

Reply:
@brucebolt I don't have a good answer to that, requires more domain knowledge than I have, but I don't at first glance see why not.

Reply:
The table under "This table contains approximate retry waiting times (click to expand)." gives a total wait time for 14 as 1d  1h 17m 36s.

Reply:
Ah, I was looking at the formula for the next backoff duration.  I've set it to 13.